packages/radix-ui-themes/src/helpers/styles.ts

```diff
@@ -0,0 +1,14 @@
+type InlineStyle = React.CSSProperties | Record<string, string | number | undefined> | undefined;
+
+// Merges CSS styles like `classNames` merges CSS classes
+export function styles(...styles: Array<InlineStyle>): InlineStyle {
```

Comment:
I get the analogy with the `classNames` package, but it took me some time to realise what this was until I saw the util and the comment.

I wonder if we should just name it `mergeStyles`?